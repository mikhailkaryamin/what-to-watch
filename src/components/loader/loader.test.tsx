import * as React from 'react';
import * as renderer from 'react-test-renderer';

import Loader from './loader'

describe(`loader`, () => {
  const wrapper = renderer
    .create(
      <Loader />
    )
  test(`should render loader`, () => {
    expect(wrapper).toMatchSnapshot();
  })
})