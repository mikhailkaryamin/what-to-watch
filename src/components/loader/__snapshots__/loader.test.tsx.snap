// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`loader should render loader 1`] = `
<div
  className="user-page"
>
  <div
    className="lds-ring"
  >
    <div />
    <div />
    <div />
    <div />
  </div>
</div>
`;
